axelor-account/src/main/java/com/axelor/apps/account/web/MoveController.java

```diff
@@ -203,15 +216,16 @@ public void deleteMove(ActionRequest request, ActionResponse response) throws Ax
   }
 
   protected void removeOneMove(Move move, ActionResponse response) throws Exception {
+    MoveRemoveService moveRemoveService = Beans.get(MoveRemoveService.class);
```

Comment:
Not sure about the changes I've done in this method but it seems weird to me to call `Beans.get(MoveService.class).getMoveRemoveService()` to get a instance of MoveRemoveService when you can just call `Beans.get(MoveRemoveService.class)`

Reply:
I agree, I think this is simply legacy code